webpack.config.ts

```diff
@@ -11,7 +12,7 @@ const config: webpack.Configuration = {
     path: __dirname
   },
   plugins: [
-    new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/)
+    new ForkTsCheckerWebpackPlugin({ silent: true })
```

Comment:
This makes it so the webpack bundle doesn't wait on typescript type checking. Brings my test bundle time from about 22s -> 15s. Errors get reported in the testserver's stdout based on the forked checker process's output.